src/main.rs

```diff
@@ -39,6 +39,9 @@ struct Args {
     help: bool,
     /// Display the current version of this software.
     version: bool,
+
+    /// Set custom output directory
+    output: Option<String>,
```

Comment:
Can you see if this can be a `PathBuf` instead? That'll protect against silly bugs that might occur in the `format!` in L184 related to path concatenation. Naturally the other arg locations would change to `&Path` to accomodate.